test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -99,7 +99,7 @@ def test_pytorch_train_dgl_gpu(pytorch_training, ec2_connection, gpu_only, py3_o
 def test_pytorch_train_dgl_cpu(pytorch_training, ec2_connection, cpu_only, py3_only):
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
     # TODO: Remove when DGL gpu test on ecs get fixed
-    if Version(image_framework_version) == Version("1.10"):
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11"):
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*"):
```